lib/lookup.json

```diff
@@ -341,7 +341,8 @@
   "node-report": {
     "prefix": "v",
     "tags": "native",
-    "maintainers": ["rnchamberlain", "richardlau"]
+    "maintainers": ["richardlau"],
+    "skip:": [">=15"]
```

Comment:
Did you mean to include this change?

Reply:
My `main` branch is behind by one commit.